doc/widgets.md

```diff
@@ -47,3 +47,31 @@ SpriteButton(
           pressedSprite: _pressedSpriteButton,
 )
 ```
+
+## FlameSpriteWidget
+
+`FlameSpriteWidget` is a widget used to display [Sprites](https://github.com/flame-engine/flame/blob/master/lib/sprite.dart) inside a widget tree.
+
+How to use it:
+
+```dart
+FlameSpriteWidget(
+        sprite: shieldSprite,
+        center: true,
```

Comment:
can you add a paragraph here explaining what centre is? it's totally not clear to me. is it vertical, horizontal, both? side note: should this be an anchor instead?

Reply:
I did the anchor thing